client/src/App.test.js

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import { render, getByText } from "@testing-library/react";
+import ContentTable from "./component/ContentTable";
+import { ExpansionPanelActions } from "@material-ui/core";
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+test("It renders successfully", () => {
+  const testArr = [{ name: "Adela", country: "Nig", search: "10" }];
+  const { getByText } = render(<ContentTable players={testArr} />);
+  const text = getByText("Adela", "Nig", "10");
+  expect(text.textContent).toBe("Adela");
 });
```

Comment:
One way we'd normally write this test would be like:
```
test("It renders successfully", () => {
  const testArr = [{ name: "Adela", country: "Nig", search: "10" }];
  const { getByText } = render(<ContentTable players={testArr} />);

  getByText("Adela"); // This will fail if it can't find the words - no need for "expect()"

  getByText(/nig/i); // Using "regex" so it doesn't matter whether the data is uppercase or lowercase

  getByText("10"); // separate the "getByText()" into multiple lines, instead of one line
});
```